docs/UserGuide.md

```diff
@@ -117,12 +117,15 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd internship application in WorkBook.
 * `find Meta` followed by `delete 1` deletes the 1st internship application within the results of the `find` command.
 
-### Clearing all entries : `clear`
+### Clearing all existing internship applications : `clear`
 
-Clears all entries from the address book.
+Clears all internship applications in the Workbook.
 
 Format: `clear`
 
+Example:
+* `clear` removes all internship applications in the workbook.
```

Comment:
I think no need to provide example of clear as it's trivial.

Reply:
I agree